.gitignore

```diff
@@ -1,3 +1,5 @@
 target/
 **/*.rs.bk
 Cargo.lock
+
+*.code-workspace
```

Comment:
That's not really an option as I sometimes _do_ want to commit this file, and ignoring it for all repositories would cause it to be forgotten about.

I've found a workaround by placing it in a nested, ignored folder, so this can be removed.